File: Assignment2.swift
/**

 Name: Tochi Iroakazi

 Class: ICS4U

 Definition: This code rums rock, paper and Scissors against the program.

 */



show("Choose a number:")

show("1 is for Rock")

show("2 is for Scissors")

show("3 is for Papper")



//makes a value betwen 3 and 1

var answer = Int(arc4random_uniform(3)+1)





var guess = Int(askForChoice("Guess", options:["1","2","3"]))



if answer == 1 {//if the computer chooses Rock.

    if guess == 1 {

        show("Its a draw, the computer chose(Rock) : " + String(answer))

    } else if guess == 2 {

        show("You Lost, the computer chose(Rock) : " + String(answer) )

    } else if guess == 3 {

        show("You won, the computer chose(Rock) : " + String(answer))

    }

};if answer == 2 {// if the computer choses 2 which is Scissors.

    if guess == 1 {

        show("You won, the computer chose(Scissors): " + String(answer))

    } else if guess == 2 {

        show("Its a draw, the computer chose(Scissors): " + String(answer))

    } else if guess == 3 {

        show("You Lost, the computer chose(Scissors): " + String(answer))

    }

};if answer == 3 {//if the computer chooses Paper 

    if guess == 1 {

        show("You Lost, the computer chose(Paper): " + String(answer))

    } else if guess == 2 {

        show("You won, the computer chose(Paper): " + String(answer))

    } else if guess == 3 {

        show("Its a draw, the computer chose(Paper): " + String(answer))

    }

}

File: Assignment2.java
/**

 * Name: Tochi Iroakazi

 * Class: ICS4U

 * Defintion: This program runs rock, paper Scissors against the computer.

 */



import java.util.Random;

import java.util.Scanner;

public class RockPaperScissors {

	public static void main(String[] args) {

		System.out.println("Pick a choice: Rock = 1, Papper = 2 and Scissors = 3 ? ");

		Scanner userinput = new Scanner (System.in);

		int userchoice = userinput.nextInt();

		Random generatenumber = new Random();

		int computerchoice = generatenumber.nextInt(3) + 1;

		switch(computerchoice) {

		   case 1:

			   //rock

			   if (userchoice == 1) { // Rock vs Rock

				   System.out.println("Draw, the computer chose(Rock) " + computerchoice);

			   }else if (userchoice == 2) { // Rock vs Papper 

				   System.out.println("Win, the computer chose(Rock) " + computerchoice);

			   }else if (userchoice == 3) { // Rock vs Scissors

				   System.out.println("Lose, the computer chose(Rock) " + computerchoice);

			   }

			   break;

		   case 2:

			   //paper

			   if (userchoice == 1) { // paper vs Rock

				   System.out.println("Lose the computer chose(Paper) " + computerchoice);

			   }else if (userchoice == 2) { // paper vs paper

				   System.out.println("Draw, the computer chose(Paper) " + computerchoice);

			   }else if ( userchoice == 3) { // paper vs Scissors

				   System.out.println("Win, the computer chose(Paper) " + computerchoice);

			   }

			   break;

		   case 3:

			   // Scissors

			   if (userchoice == 1) { // Scissors vs Rock

				   System.out.println("Win,the computer chose(Scissors) " + computerchoice);

			   }else if (userchoice == 2) { // Scissors vs paper

				   System.out.println("Lose, the computer chose(Scissors) " + computerchoice);

			   }else if ( userchoice == 3) { // Scissors vs Scissors

				   System.out.println("Draw, the computer chose(Scissors) " + computerchoice);

			   }

			   break;

		}

	}



}
